scripts/manifests/model_registry_resource/model_registry_resource.yaml

```diff
@@ -10,14 +10,9 @@ metadata:
 spec:
   grpc: {}
   rest: {}
-  istio:
-    gateway:
-      grpc:
-        tls: {}
-      rest:
-        tls: {}
+  oauthProxy: {}
   mysql:
-    host: model-registry-db.rhoai-model-registries.svc.cluster.local
+    host: model-registry-db.$MR_NAMESPACE.svc.cluster.local
```

Comment:
If the service is in the same namespace, you could use just the hostname/servicename too.

Reply:
Oh, I did not know this. But I do already need to update this file for the port place holder, so changing the namespace was also done in the same sed command.